Draft for Audio Support

moritzstaffel opened the pull request.

moritzstaffel committed:
Speaker Geometry

moritzstaffel committed:
Create proposal.md

moritzstaffel committed:
Update proposal.md

moritzstaffel committed:
Merge branch 'next' into audio-proposal

moritzstaffel committed:
Add Properties of Speaker

moritzstaffel committed:
Add listenting plane

moritzstaffel committed:
Update proposal.md

moritzstaffel committed:
Update proposal.md

moritzstaffel committed:
Update proposal.md

moritzstaffel commented:
I have added the information so fro that we researched. What is missing is how we handle attachments, hanging material and other stuff.
Also we need to get more input on the things that we researched.

petrvanekrobe commented:
additional items needed for audio support:

need to support a range of angles for the sound angle.  for their speakers they have a set of angles within the range that they support.  for others they can support any angle in that range

for sound angle, need to support asymetric dispersion with a different angle possible in each direction

also need to support rotation of the sound angle

max SPL is not always in dB.  There are different dB formats like a weighted dB.

for mvr, need different material types to support reflectiveness for the audio analysis

ILane-VW commented:
Received the following information about different formats to support for sound level:

Here is a table with different excitation signals and weightings.
Excitation Signals and weightings are independent from each other and can be used in any combination.

| Excitation Signal                                        | Frequency Weighting | Unit                         | Time Weighting | Unit                    |
| :--------------------------------------  | :-------------------- | :------------------ | :--------------- | :--------------- |
| Pink noise                                                  | None                            | DBZ, dB or DBSPL | Fast                     | F                          |
| IEC60268 "Program Simulation Noise"  | A-Weighting                | dBA                         | Slow                    | S                          |
| EIA426B "Loudspeaker test signal"        | C-Weighting               | dBC                         | Leq                      | Leq, Laeq, LCeq |
| AES75 "M-Noise"                                      |                                     |                                  | Peak                   |                             |

petrvanekrobe commented:
GDTF: For sound angles, we will not do a range - we will define the horizontal Left, horizontal Right, vertical Up, vertical Down, and Rotation. 
MVR: Define materials by names (string).

petrvanekrobe committed:
Update the proposal based on latest feedback/comments

petrvanekrobe committed:
Add initial Amplifier specification

klinzey committed:
Define ListeningPlane node and its attributes

Add definition for ListeningPlane and its attributes.

klinzey committed:
Revise speaker and MaxSPL attributes in gdtf-spec

Updated speaker attributes and added MaxSPL attributes.

klinzey commented:
From the MVR standpoint I think we can use overrides and adjust  min X, min Y, max X, max Y to the appropriate values when the speaker has an adjustable beam? I don't think we need an additional field in the MVR to store that data. Do we need a flag in the GDTF to mark it as adjustable or can we just use the fact that Min<>Max signifies that it is adjustable?

klinzey committed:
Define Speaker node with attributes and children

Added a new Node Definition for Speaker with attributes and child nodes.

petrvanekrobe committed:
Adjusting attributes and their definitions, adding some todo items

JGaudin commented:
About the current description :

Sound rotation angle :
This is confusing, and is generally driven by the horn rotation of the loudspeaker. Some loudspeakers have rotatable horns, the horn provides the directivity in the high frequencies. Should be more explicit to a sound person.

M-Noise in AES-75 context is now called Music Noise suggesting AES75MusicNoise

Max SPL weighting : 
Generally accepted that "none" = Z

Max SPL :
Max SPL = dB SPL there is no other unit available, but it is described using the above weighting.

For example if an A or C filter is applied to the signal, the value will be written as 200 dB SPL A

Averaging time (time weighting): 
Fast (F), Slow(S) and Peak(Pk) are defined. L is the "Length of the average and needs to be specified in minutes.

For example 200 dB SPL LAeq, 30 is A weighted signal averaged over 30 minutes.
Which I believe we could write as 200 dB SPL L A, 30
For peak values we'd have 200 dB SPL Pk Z or sometimes (Z)

Impedance is not used for self-powered loudspeakers. 
Assuming that non-relevant fields can stay empty ?

About Sound Angle : 
Loudspeakers are not producing energy forward with left and right boundaries, they would produce a bubble of energy in from the loudspeaker connected to it, that would behave different a various frequencies. The angle coverage is an average of all these and we should be careful of the future use of these values : it does not describes the power  distribution of the energy in this case.

Line-arrays have proportional vertical angles, meaning they vary with frequency. Low frequencies are very wide, higher frequencies very narrow, up to a factor 10 sometimes.
We should have a field for "Proportional" or "Variable" that could also covers loudspeakers that have variable horns, like the L-Acoustics K2, for example.

JGaudin commented:
Proposed new fields : 

Internal amplifier (or Self-Powered) - Yes / No
External Power Supply - Yes / No
External Power Supply Voltage - value in VDC
External Power Supply compatible models - Text field / link (?)
Power, Maximum Long Term Continuous - Watts
Power, Idle - Watts
Operating voltage range Minimum - Voltage
Operating voltage range Maximum - Voltage
Operating voltage frequency - Hz
AC Power connector -
DC Power connector - 
Power loop output - Yes / No
Audio Input - Analog, AES3, Dante, MILAN, AES67, other
Audio Connectors - XLR3, XLR5, Phoenix, Ethernet
Audio loop output - Analog, AES3, Dante, MILAN, AES67, other
User DSP - Yes / No
Latency - ms
IP rating
Rigging accessories - U-Braket, Single arm, Rotoclamp, Yoke,  Eyebolt
Max SPL Reference RMS input level - dBV

petrvanekrobe committed:
Further definitions for Audio proposal

- add file based definition for ListeningPlaneHeatMaps
- add HangingPoint
- add SpeakerBumper to StructureType

petrvanekrobe committed:
Extend the definition for the ListeningPlaneHeatMaps

petrvanekrobe committed:
Remove old ListeningPlane example

klinzey committed:
Refine ListeningPlaneHeatMap and Face node definitions

Updated tables to include attribute value types and added Units attribute to ListeningPlaneHeatMap.

klinzey committed:
Revise speaker attributes and add XML example

Updated speaker attributes table and added examples for clarity.

petrvanekrobe committed:
Fix table

klinzey committed:
Revise speaker geometry and SPL attributes

Updated speaker geometry attributes and added new SPL attributes. Clarified the purpose of the <Speaker> node and provided an example of a node definition.

klinzey committed:
Merge pull request #312 from mvrdevelopment/GDTF-Speaker-Sample

Revise speaker attributes and add XML example

petrvanekrobe committed:
Add Amplifier object

klinzey committed:
Delete Extra Color node definition from mvr-spec.md

Removed Extra Color node definition from listening plane heat map
